### ISSUE TITLE
Add PoissonRegressor

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#74]
Add PoissonRegressor model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference PoissonRegressor Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/PoissonRegressor.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.PoissonRegressor.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an PoissonRegressor as a potential best fit model.

**Comment by melan96** [comment]
Hi @Thilakraj1998 I will go through this issue. Can you assign me to this?
